packages/utils/src/utils/index.js

```diff
@@ -356,6 +356,14 @@ export function getGenerateNamePrefixForRerun(name) {
   return `${root}${rerunIdentifier}`;
 }
 
+export function getGenerateNamePrefixForNewRun(name) {
+  let generateName = name;
+  if (!name.endsWith('-')) {
+    generateName = `${name}-`;
+  }
+  return `${generateName}`;
```

Comment:
no need to use template here:
```suggestion
  return generateName;
```